### PR TITLE
0.2.0.1

### PR DESCRIPTION
**Pull request by jhiker**
As part of our work setting up LaunchDarkly provider, we would like to merge this into main.

### CODE REVIEW

**Review by maxveldink — DISMISSED** [review]

File: openfeature-sdk-sorbet.gemspec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Gem::Specification.new do |spec|
   spec.name = "openfeature-sdk-sorbet"
-  spec.version = "0.2.0"
+  spec.version = "0.2.0.1"
```
Comment (mcverry-at-justworks) [inline openfeature-sdk-sorbet.gemspec:5]: @jhiker this should be `"0.2.1"`

Comment (mcverry-at-justworks) [reply openfeature-sdk-sorbet.gemspec:5]: see: https://github.com/maxveldink/openfeature-sdk-sorbet/releases/tag/0.2.1

**Comment by maxveldink** [comment]
@jhiker Sorry, I gave this a quick approval yesterday, but I believe the already published `0.2.0.2` is the one you want for the loosened Hash type (which might already be pulled into CW).

I released `0.2.0.2` and `0.2.1` at the same time, both of which included an important bug fix. The only difference is that `0.2.0.2` has the loosened Hash type that you're trying to add in this PR. `main` branch includes a lot of unreleased work (that I need to get a jump on, thanks for the kick in the pants!).

I did a poor job publicizing this because I thought these would be quick-lived versions, but alas.

In short, I think you already have this behavior in CW. I'd be happy to chat more if you're not seeing it, and I'll be looking to release a `0.3.0` soon. It will have a few breaking changes.

**Comment by mcverry-at-justworks** [comment]
Ah gotcha. Thanks @maxveldink for the clarification. My local CW was pulling down `0.2.0`, but we will need `MultipleSourceProvider` to pass `context`. This was fixed in `0.2.1`. (This fix is not currently in `main` yet).  My intent with requesting this PR was a low-pri maintenance merge to bring that fix back into `main`. 

Should we add https://github.com/justworkshr/openfeature-launchdarkly-provider/ before you have time to upgrade to `0.3.0`,  I can bump the version in CW's gemfile to make it explicit during installs. 

Thanks again @maxveldink and @jhiker
